static/app/views/alerts/rules/metric/details/metricChartOption.tsx

```diff
@@ -355,6 +403,72 @@ export function getMetricAlertChartOption({
       });
   }
 
+  if (anomalies) {
+    const anomalyBlocks: any[] = [];
```

Comment:
anomalyBlocks could use a better type
```suggestion
    const anomalyBlocks: any[] = [];
```

Reply:
Oof ok.
this was way too hairy.

not gonna bother typing this 😬

Reply:
it should be `MarkAreaComponentOption['data']`